src/materials/MeshBasicMaterial.js

```diff
@@ -54,6 +54,7 @@ function MeshBasicMaterial( parameters ) {
 	this.aoMapIntensity = 1.0;
 
 	this.specularMap = null;
+	this.specularStrength = 1;
```

Comment:
`this.color + this.map`
`this.opacity + this.alphaMap`
`this.emissive + this.emissiveIntensity + this.emissiveMap`

etc. etc. the pattern is, anything that is *this.something***Map** comes with some modulating color or factor. 

**But completely arbitrary, `specularMap` doesn't.** 

*(ignore the fact that specularMap shouldnt even be present on this material)*

As a user, i am aware of the properties of a "texture". I know that it is in a certain format and that it is normalized/clamped. Ergo, all i know is that the values in those textures will be 0-1. 

Since it is possible to use a texture that has only one value, and [since people are using it](https://github.com/mrdoob/three.js/issues/10808#issuecomment-280398838), i figured, give the users a shorthand the same way other maps kinda work.